contrib/cirrus/setup.sh

```diff
@@ -10,6 +10,17 @@ source $(dirname $0)/lib.sh
 
 req_env_vars OS_RELEASE_ID OS_RELEASE_VER GOSRC IN_PODMAN_IMAGE
 
+_EOL=20230601
+if [[ $(date +%Y%m%d) -ge $_EOL ]]; then
+    die "As of $_EOL this branch is probably
+no longer supported in RHEL 8.7/9.1, please
+confirm this with RHEL PM.  If so:
```

Comment:
I'd suggest "Program Manager" over "PM" as that could also be "Product Manager" and they generally don't always know the RHEL deadlines like program managers do.  Currently that is Laurie Friedman

Reply:
Hmmm, darn.  I'm not disagreeing...it's just that this is the wording already in-place for a few podman branches.  I just copy-pasted the message :frowning_face: 

I'll fix it here and https://github.com/containers/buildah/pull/4771 but I'd rather not track down and fix the already-merged podman PRs - simply out of laziness :grin: